app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -108,6 +114,27 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
         }
     }
 
+    private class DrawOnce(private val view: View) : ViewTreeObserver.OnDrawListener {
+        private val uiHandler = Handler(Looper.getMainLooper())
+
+        override fun onDraw() {
+            // I verified that this frame is the one that draws the EditBookmark by adding thread.sleep
+            // into uiHandler.post.
+            uiHandler.post {
+                view.viewTreeObserver.removeOnDrawListener(this)
+            }
+
+            val now = SystemClock.elapsedRealtime()
+            android.util.Log.e("lol", "displayed ${now - MenuButtonClick.initialMillis}")
```

Comment:
NIT: Not sure if we should leave the log there 😄

Reply:
We're not landing this code so I think it's fine.